src/app.postcss

```diff
@@ -16,7 +16,7 @@
   --primary: hsl(1, 70%, 80%);
   --primary-text-color: var(--black);
   --secondary: hsl(220, 66%, 21%);
-  --red30: hsl(4.02, 100%, 46.87%, 0.3);
+  --color-selection: hsl(4.02, 100%, 46.87%, 0.3);
```

Comment:
let's call this something meaningful such as --highlight and let's use red30 to set it
so ```--highlight: var(--red30);```

You can implement something similar for the rest of the colors